Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs

```diff
@@ -239,7 +239,7 @@ public void Test()
 }
 ";
 
-			VerifyDiagnostic(template, DiagnosticResultHelper.Create(DiagnosticId.NoUnnecessaryStringFormats));
+			VerifyDiagnostic(template, DiagnosticId.NoUnnecessaryStringFormats);
```

Comment:
Putting one constant in this file wouldn't hurt here

Reply:
I don't understand @ynsehoornenborg